README.md

```diff
@@ -146,106 +274,151 @@ You can add this on the same HTML file you added the translatable strings before
 </form>
 ```
 
-Add the following route in your Project's routes, so that the `set_language` hook shown above will work when submitting the form.
-```python
-from django.conf.urls import url, include
-
-urlpatterns = [
-    ...,
-    url(r'^i18n/', include('django.conf.urls.i18n')),
-]
-```
-Last, add `'django.middleware.locale.LocaleMiddleware'` in your `settings.MIDDLEWARE` to enable the functionality.
-
 Now you can test the language picker. Each string will be shown translated in the current language.
 
 If a translation is not available on Transifex, the source string will appear instead by default.
 This behavior is configurable by defining a different [missing policy](#missing-translations).
 For example, you can choose to show dummy content instead of the source string, a method often referred to as “pseudo-localization”.
+
 This way, you can test the UI and allow strings that have not been translated to stand out.
 
-```python
+```django
 TRANSIFEX_MISSING_POLICY = 'transifex.native.rendering.PseudoTranslationPolicy'
 # _t("Hello, friend") -> returns "Ȟêĺĺø, ƒȓıêñđ"
 ```
 
-### Detailed usage
+#### Use in Templates and Views
 
 You can use the toolkit both inside Django templates as well as inside views.
 
-#### Internationalization in template code
+##### Internationalization in Template Code
 
 First of all, near the top of every template in which you want to include
+
 localized content, add the following template tag:
 
-```
+```django
 {% load transifex %}
 ```
 
-Translations in Django templates use one of two template tags, `{% t %}` and
-`{% ut %}`. They translate strings or variables that contain strings. The
-strings themselves can be constant, or they can contain variables which can be
-resolved either from the parameters of the template tags or the context of the
-template. The difference between the two tags will be addressed later, under
-[XML escaping](#XML-escaping).
+Translations in Django templates use one of two template tags, `{% t %}` and `{% ut %}`. They translate strings or variables that contain strings. The strings themselves can be constant, or they can contain variables which can be resolved either from the parameters of the template tags or the context of the template. The difference between the two tags will be addressed later, under [XML escaping](#XML-Escaping).
 
 ```html
```

Comment:
we can use django syntax highlighting here as well, now that we know it works.

Reply:
Done